ui/components/MesheryPatterns.js

```diff
@@ -102,6 +102,20 @@ function CustomToolbar(onClick, urlOnClick) {
         <label htmlFor="url-upload-button">
           <URLUploader onSubmit={urlOnClick} />
         </label>
+        <label htmlFor="create-pattern">
+          <Tooltip title="Create Pattern">
+            <IconButton
+              aria-label="Create Pattern"
```

Comment:
Duplicate label. We don't need those. Please remove `<label>` from the nodes (all).

Reply:
Ok.

Reply:
I couldn't remove the `<label>` tag from the the upload button. It was being used by the `input` tag. Removing the label tag the button did not work as desired.